### PR TITLE
Blog post with release 1.4 plan

### PR DESCRIPTION
**Pull request by pbochynski**

### CODE REVIEW

File: content/blog-posts/2019-07-23-release-1.4-plan/index.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+We are working also on the better application registration process. We started new subproject - Compass, that is still in the Kyma Incubator, but you will get the preview of the new API and UI in the new release. [Compass](https://github.com/kyma-incubator/compass) consists of components that provide a way to register, group and manage your applications across multiple Kyma runtimes. Using Compass, you can control and monitor your application landscape in one central place. It will be enabled in Kyma as an experimental feature activated by the feature toggle.
+
+Next Kyma release will have more flexible AddOns. You will be able to activate AddOns in selected namespaces only, and the publishing process will be much simpler, as you can host them directly from GitHub.
+
+Here are some other things that will come in Kyma 1.5 Kyoto:
```
Comment (magicmatatjahu) [inline content/blog-posts/2019-07-23-release-1.4-plan/index.md]: **1.4** not **1.5**

File: content/blog-posts/2019-07-23-release-1.4-plan/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Namespace view improvements, like better labeling and filtering, or health status of deployed workloads.
+- Kiali in the latest stable version and with Single Sign-On
+- Upgrade to Istio 1.2.2
+- The backup procedure is configurable as part of the installation
+- Conversion and validation service for AsyncAPI specifications
+- Experimental standalone (CRD only) Kyma Function controller module with KNative implementation
+
+You can find all the items planned for the 1.4 release in our [Zenhub](https://app.zenhub.com/workspaces/kyma---all-repositories-5b6d5985084045741e744dea/reports/release?release=5cd88a49145f41296ed5194a).
```
Comment (magicmatatjahu) [inline content/blog-posts/2019-07-23-release-1.4-plan/index.md]: Maybe also add link to our roadmap? 😄

File: content/blog-posts/2019-07-23-release-1.4-plan/index.md (hunk 1 above)
Comment (tomekpapiernik) [reply content/blog-posts/2019-07-23-release-1.4-plan/index.md]: fix'd

File: content/blog-posts/2019-07-23-release-1.4-plan/index.md (hunk 2 above)
Comment (tomekpapiernik) [reply content/blog-posts/2019-07-23-release-1.4-plan/index.md]: Done :)

**Pull request merged**